packages/api/amplify_api/test/graphql_test.dart

```diff
@@ -161,13 +162,16 @@ class MockAmplifyAPI extends AmplifyAPIDart {
   });
 
   @override
-  MockWebSocketBloc getWebSocketBloc({String? apiName}) {
+  MockWebSocketBloc getWebSocketBloc(EndpointConfig endpoint) {
```

Comment:
I think unit tests are failing bc this needs to be renamed to match renamed util.